pom.xml

```diff
@@ -124,15 +124,15 @@
             <artifactId>spring-cloud-spring-service-connector</artifactId>
         </dependency>
 
-        <!-- https://mvnrepository.com/artifact/org.springframework.cloud/spring-cloud-sleuth-zipkin -->
         <dependency>
-            <groupId>org.springframework.cloud</groupId>
-            <artifactId>spring-cloud-sleuth-zipkin</artifactId>
+            <groupId>org.springframework.amqp</groupId>
```

Comment:
is this required? I would have thought this would already be a part of spring-cloud-starter

Reply:
Adding this dependency fixed the travis build error. (Check the "added spring security dependency" commit)